### DISCUSSION TITLE
[Feature] Show mail count configured individually per tab

### DISCUSSION THREAD

**Discussion by joomlafun** [RealRaven2000/QuickFolders#502] [Ideas]
Hi,

As you know, presently you can by with a global option show the count of mails on tabs and then individually hide it(using the tab properties)

Could it possible to have in the tab property an option added to show the count of mail or not for this specific tab when the global option is disabled?

Case example : I place certain mails that I have to answer in a folder and if the count is shown I know that I have to answer the emails
 
And if I rely on the read/unread status to know there are email present in the folder it is unreliable and prone to make mistakes forgetting to apply those status

**Comment by RealRaven2000** [comment]
> Could it possible to have in the tab property an option added to show the count of mail or not for this specific tab when the global option is disabled?

That seems very specific... first I thought only to display the **count of unread emails**  there, but do you mean the count of **all contained emails**?

Does this mean that the folder is usually empty? It just seems to be a very specialized workflow, not sure that many users would benefit from it, so please talk me through how you would make this useful for your work. I want to avoid overburdening QuickFolders with too many features that may only benefit a very small number of people with very specialized workflows. I usually fail in this because it is already such a vastly complicated tool.

**Comment by RealRaven2000** [comment]
PS: I did an edit for clarity, so please read previous comment on Github (it won't tell you the edit via email)

**Comment by joomlafun** [comment]
Hi there,

Thank you for your feedback.

After giving it a second thought it is manageable for me just to activate the count by default and then individually disable it on all the folders to show it only on the one I want it to be shown.

But if you want to know why I tend to work that way is that if you want to estimate a quantity of emails from specific demands that you need to answer, a dedicated folder with its count is useful. Why not relying on the count of unread emails count? Because this count is unreliable, u may forget to set it back once you read an email etc.. 

So, all good, thank you for your time.
Great extension, keep up the good work

**Comment by RealRaven2000** [comment]
> But if you want to know why I tend to work that way is that if you want to estimate a quantity of emails from specific demands that you need to answer, a dedicated folder with its count is useful. Why not relying on the count of unread emails count? Because this count is unreliable, u may forget to set it back once you read an email etc..

still trying to understand, if you do not empty this folder it will just be an ever increasing number, do you keep that in mind, or how do you know you have a new workload there?

**Comment by joomlafun** [comment]
Each time a mail from this folder is "treated", I just archive it or place it in another folder.

I must say that I work with the "zero inbox method", have folders with delay of answer "today", "this week etc" ; if I see a count of 5 on the this week folder, I know I still have to take care of this quantity of item within the current week. On Sunday I will make sure that this folder is empty.

**Comment by jaysmithgit** [comment]
OFF TOPIC:

Hi [joomlafun](https://github.com/joomlafun)...

I had to laugh (in a good way). I very much admire your "zero inbox method". I wish I could make such a system work, but I get more REAL emails into my inbox every day than I could actually accomplish in two or three days. Yes, I do have multiple inboxes -- when the main/current/real inbox has over 2000 emails in it, I move them aside to another "inbox" named for the date range of the contents. I only keep real emails that genuinely need action. (The spam has already been filtered out by a great service [Duocircle].) Obviously I am involved in too many projects and subjects, and I have been unsuccessful in getting myself cloned. Sometimes I wish for the days before email made more work for me.

> Each time a mail from this folder is "treated", I just archive it or place it in another folder.
> 
> I must say that I work with the "zero inbox method", have folders with delay of answer "today", "this week etc" ; if I see a count of 5 on the this week folder, I know I still have to take care of this quantity of item within the current week. On Sunday I will make sure that this folder is empty.

**Comment by RealRaven2000** [comment]
> I had to laugh (in a good way). I very much admire your "zero inbox method". I wish I could make such a system work, but I get more REAL emails into my inbox every day than I could actually accomplish in two or three days. Yes, I do have multiple inboxes -- when the main/current/real inbox has over 2000 emails in it, I move them aside to another "inbox" named for the date range of the contents. I only keep real emails that genuinely need action.

Yes, the **Zero Inbox** approach is a valid working method. In my case, most of my email doesn't stay in the inbox but is auto-filtered to the correct subscription folders, so my daily 100-150 mails end up staying within the single digits in the inbox. I usually manage to move them from the Inbox quickly (as they should be in topical folders - such as "Addon X/Support", or they are just unimportant, like stuff from mailing lists "plugins/vendor X") 

My main focus folders are tabs and will show up bold with a number of unread mails, so I handle them (and the few remaining unread mails in the inbox) as my first priorities; then I usually browse the rest by using the "N"ext unread mail to quickly browse all other stuff, which doesn't really need more than 1-5 seconds / mail. If I want to leave a folder with unread mails for later I use the Shift+J (jump to next unread folder). If there is a really important email I need to reply and cannot do it quickly I open it in a tab so I don't forget about it.

**Comment by joomlafun** [comment]
@RealRaven2000 Cool method for dealing with unread emails, thank you for sharing.
Would you have a method for dealing/finding quickly emails that you have sent and that are left with no answer from the recipient for more than 2 weeks for example?

I like ticketing system for that when you send a message from the ticket then the status becomes pending ; u can then search for all pending tickets for x days/week

**Comment by RealRaven2000** [comment]
Not sure, you might try a combination of Status isn't replied and age in days > 10 days, or something along those lines?

![image](https://github.com/user-attachments/assets/710f1eaa-c33a-4173-b316-0bc8ff465d5e)

**Comment by joomlafun** [comment]
Well actually this searches the emails that you did not reply to.

What I need is to track all sent mails which did not get an answer since > ..

**Comment by RealRaven2000** [comment]
I converted this issue into a Discussion. we might yet still make a new feature request, but need more clarity and some valid work flows that (a reasonable number of) users could use.

**Comment by joomlafun** [comment]
@RealRaven2000 
It is kind from you. I always understand that a provider needs to evaluate the demand. It hard to know sometimes.. Maybe each year, a few months before Christmas , there could be a poll among users to see the most wanted features :)
